packages/adminrouter/extra/src/lib/cache.lua

```diff
@@ -621,6 +611,14 @@ local function periodically_refresh_cache(auth_token)
     else
         ngx.log(ngx.INFO, "Created initial recursive timer for cache updating.")
```

Comment:
@ahairape Let's please try to adjust the log messages as well, now that there are no recursive timers :) Thanks!

Reply:
Thanks for the hint, @vespian. I have fixed it.